notebooks/vertex_genai/solutions/vertex_llm_tuning.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>It seems <code>us-central1</code> is also another option now (but using different environment)</p><p> </p><p>https://cloud.google.com/vertex-ai/docs/generative-ai/models/tune-text-models</p><p> </p><p>Also, if the training is in <code>europe-west4</code> , maybe the bucket should also be in the same region as a best practice? (although the data is small in this case)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Great! It seems to work with all in us-central1. Done!</p>

Reply:
<p>Unfortunately, the LLM tuning part of the pipeline fails if not set to europe-west4 (as opposed to us-central1). This is certainly temporary, but will keep it this way for now</p>